src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -29,15 +29,27 @@ class DebugCommand extends Command
     private $twig;
 
     /**
-     * {@inheritdoc}
+     * @param Environment $twig
      */
-    public function __construct($name = 'debug:twig')
+    public function __construct($twig = null)
     {
-        parent::__construct($name);
+        parent::__construct();
+
+        if (!$twig instanceof Environment) {
+            @trigger_error(sprintf('Passing a command name as the first argument of "%s" is deprecated since version 3.4 and will be removed in 4.0. If the command was registered by convention, make it a service instead.', __METHOD__), E_USER_DEPRECATED);
+
+            $this->setName(null === $twig ? 'debug:twig' : $twig);
```

Comment:
Yeah.. to me that should be `null !== $name` as well :) https://github.com/symfony/symfony/blob/5129c4cf7e294b1a5ea30d6fec6e89b75396dcd2/src/Symfony/Component/Console/Input/InputInterface.php#L27

If you insist i change it, just saying im preserving previous behavior :)